cypress/integration/core/date-picker.spec.ts

```diff
@@ -400,22 +410,27 @@ describe('Date Picker', () => {
       });
   });
 
-  it('should emit changed event with error on invalid character paste', () => {
+  it('should not allow invalid characters to be pasted', () => {
     const details = [];
     cy.get('dso-date-picker').then(datePicker => {
-      datePicker.get(0).addEventListener('dateChange', (event: CustomEvent) => details.push(event.detail))
+      datePicker.get(0).addEventListener('dateChange', (event: CustomEvent) => details.push(event.detail));
     });
 
-    cy
+    const input = cy
```

Comment:
Het opslaan van verwijzingen naar elementen moet je niet in de JS/TS wereld doen. Cypress is asynchroon, vandaar dat je op regel 427 die dubbele `.then()` nodig hebt om bij de juiste value/state te komen.

Als je een verwijzing wilt opslaan kan je dat het beste met `.as('input')` doen. Die kan je dan weer opvragen met `.get('@input')`.